<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductPurchase</class>
 <widget class="QWidget" name="ProductPurchase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProductPurchase</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>971</width>
     <height>720</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>969</width>
      <height>718</height>
     </rect>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>0</y>
       <width>961</width>
       <height>600</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="clearShoppingCart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>630</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>清空购物车</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Back">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>630</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>返回主界面</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AddCommodity">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>630</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>添加商品</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removeCommodity">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>630</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>移除商品</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ShoppingCart">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>630</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>显示购物车</string>
     </property>
    </widget>
    <widget class="QPushButton" name="searchCommodity">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>630</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>搜索商品</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
